src/common/API/storageAPI.tsx

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const getQueueNumber = async (network: string): Promise<any> => {
+  const url = `api${network === "homestead" ? "" : `-${network}`}.tradetrust.io/storage/queue`;
```

Comment:
not sure if you want to do it in this story but i think there's a story for configuring this in the configuration file - just something to take note of

but i don't think the endpoint should be hardcoded in the function body

Reply:
ok i understand, so for now can we have this hardcoded, until we have the story for the configuration file? 
ill add a note inside to indicate the hardcoded url. 
is that ok?

Reply:
ok